app/scripts/app.js

```diff
@@ -61,6 +61,9 @@ angular
           return HomePagePreferenceServiceProvider.$get().getHomePagePath();
         }
       })
+      .when('/checkMobileEnabled', {
```

Comment:
while this is good for testing, I don't think we should keep it

Reply:
agreed, it is there just for verification/testing